exercises/binary-search/.meta/solutions/binary_search.rb

```diff
@@ -2,15 +2,15 @@ class BinarySearch
   attr_reader :list
```

Comment:
If `list` is not longer tested, this should probably be private, however, we are generating the exercise tests, the PR isn't about the "proof of exercise", so perhaps this is a different PR.